src/spy.js

```diff
@@ -0,0 +1,20 @@
+export function spy(cb) {
```

Comment:
I would say a normal use case is to spy on a function that you don't care about the return value:
```ts
const fn = spy(() => undefined);
```
It would be nice if this function could have that behavior as default of no function is given:
```suggestion
export function spy(cb = () => void 0) {
```